.github/workflows/auto_tag.yml

```diff
@@ -136,18 +135,10 @@ jobs:
             rc*) $SET_STABILITY rc ;;
           esac;
 
-      - name: Patch parent version for OSS
-        if: github.event.repository.name == 'oss'
-        run: jq '.require["ibexa/admin-ui-assets"] |= "${{ inputs.version }}"' composer.json | sponge composer.json
-      - name: Patch parent version for Content
-        if: github.event.repository.name == 'headless'
-        run: jq '.require["ibexa/oss"] |= "${{ inputs.version }}"' composer.json | sponge composer.json
-      - name: Patch parent version for Experience
-        if: github.event.repository.name == 'experience'
-        run: jq '.require["ibexa/headless"] |= "${{ inputs.version }}"' composer.json | sponge composer.json
-      - name: Patch parent version for Commerce
-        if: github.event.repository.name == 'commerce'
-        run: jq '.require["ibexa/experience"] |= "${{ inputs.version }}"' composer.json | sponge composer.json
+      - name: Patch parent version for Headless
+        run: |
+          jq '.require["ibexa/headless-assets"] |= "${{ inputs.version }}"' composer.json | sponge composer.json
+          jq '.require["ibexa/oss"] |= "${{ inputs.version }}"' composer.json | sponge composer.json
```

Comment:
not sure but this package is not in OSS, it start from headless so it little bit weird to see OSS here.

Reply:
> not sure but this package is not in OSS, it start from headless so it little bit weird to see OSS here.

But `ibexa/oss` is in this package. That's the line that gets patched: https://github.com/ibexa/headless/blob/4.6/composer.json#L19.